package.json

```diff
@@ -46,8 +46,8 @@
   },
   "scripts": {
     "start": "magic server",
-    "dist": "npm run deps && magic make dist/desktop dist/touch-pad dist/touch-phone",
-    "build": "npm run deps && magic make desktop.examples desktop.tests desktop.docs touch-pad.examples touch-pad.tests touch-pad.docs touch-phone.examples touch-phone.tests touch-phone.docs *.bundles/*",
+    "dist": "npm run deps && magic make dist/desktop dist/touch",
+    "build": "npm run deps && magic make desktop.examples desktop.tests desktop.docs touch.examples touch.tests touch.docs *.bundles/*",
```

Comment:
why should we build `tests`?
> `*.bundles/*`

what bundles you want to build here?
Can we write this command more compact? For example, `npm run deps && magic run examples docs && magic make *.bundles/*`